src/renderer/partials/About.vue

```diff
@@ -41,21 +43,11 @@ export default {
       },
       aboutProps: [{
         items: [{
-          image: '/static/img/data-curator-128.png',
-          height: '48px'
-        },
-        {
-          label: 'Data Curator',
-          style: {
-            fontSize: '20px',
-            color: '#0ca831'
-          }
-        },
-        {
-          label: 'Create usable open data'
+          image: '/static/img/data-curator-banner.png',
+          link: 'https://github.com/ODIQueensland/data-curator/tree/master/'
         },
         {
-          label: '0.1.0'
+          label: 'version: 0.2.2'
```

Comment:
can this be automated?

Reply:
The version is automated: will put the image and link in new about partial.